internal/connector/test/integration/cleanup/main.go

```diff
@@ -63,24 +65,28 @@ func main() {
 		if err != nil {
 			panic(err)
 		}
+		clientsKafkaSre, err := kcClientKafkaSre.GetClients(accessTokenKafkaSre, last, 100, "")
```

Comment:
This will try to clean all the clients created. it is okay for keycloak container, but gitlab test run agains mas-sso stage. That will be the issue.